pyproject.toml

```diff
@@ -64,9 +64,11 @@ pytest = "^7"
 pytest-cov = "^4.0.0"
 pytest-depends = "^1.0.1"
 mkdocs = "^1.4.3"
-mkdocs-material = "^9.1.15"
-mkdocstrings = {extras = ["python"], version = "^0.22.0"}
 cairosvg = "^2.7.0"
+mkdocs-material = "^9.1.15"  # insiders fork installed out-of-band in docs/setup_insiders.sh
+mkdocstrings = { version = ">0.20,<1", extras = ["python"] }
+mkdocs-git-committers-plugin-2 = "^1.1.2"
+mkdocs-git-revision-date-localized-plugin = "^1.2.0"
```

Comment:
To be clear, these are not `insiders` features, correct? A `poetry install` will still work, even for someone without `kolenaIO` access?

Reply:
Neither of these repos are private (note that the pins don't point anywhere special, and will thus just install the packages from PyPI).

`mkdocs-git-revision-date-localized-plugin` is a non-insider feature that is enabled in `mkdocs.yml` via the `git-revision-date-localized` block in `plugins`.

The other one is an insider feature, which is only enabled when the `git-committers` block is present (see `mkdocs.insiders.yml`). However, since it can be installed publicly, I think it's preferable to have it as a part of the main dev dependency specification in `pyproject.toml`, rather than install as a part of `setup_insiders.sh`.